tests/contrib/monitors/mixins/test_validation.py

```diff
@@ -0,0 +1,214 @@
+import pytest
+
+from spidermon.contrib.monitors.mixins import ValidationMonitorMixin
+from spidermon.contrib.scrapy.monitors import BaseScrapyMonitor
+from spidermon.data import Data
+
+
+stats = {
+    'spidermon/validation/fields': 100,
+    'spidermon/validation/fields/errors': 20,
+    'spidermon/validation/fields/errors/missing_required_field': 15,
+    'spidermon/validation/fields/errors/missing_required_field/field2': 5,
+    'spidermon/validation/fields/errors/missing_required_field/field3': 10,
+    'spidermon/validation/fields/errors/': 10,
+    'spidermon/validation/items': 10,
+}
+
+
+class DummyValidationMonitor(BaseScrapyMonitor, ValidationMonitorMixin):
+    def __init__(self, stats, methodName="runTest", name=None):
+        super(DummyValidationMonitor, self).__init__(methodName, name)
+        self.data = Data({'stats': stats})
+        self.new_behavior = True
```

Comment:
This is related to your comment, so it wold be nice to have a check for `new_behavior = False`.
Maybe, creating pytest fixtures for `new_behavior = False` and `new_behavior = True`.
It looks like we can call `__init__` in the mixin https://stackoverflow.com/questions/6098970/are-mixin-class-init-functions-not-automatically-called

Reply:
_old() tests set new_behavior = False and test it, but I agree it can be useful to have them more separate.